__tests__/example.test.ts

```diff
@@ -0,0 +1,54 @@
+type GetTestingRequest = {
```

Comment:
This entire file is only example. You can delete it. It shouldn't be on production